capsule/src/main/java/Capsule.java

```diff
@@ -5459,7 +5460,8 @@ static Path createPathingJar(Path dir, List<Path> cp) {
         try {
             final Path absolutePath = dir.toAbsolutePath();
             final List<String> paths = createPathingClassPath(absolutePath, cp);
-            final Path pathingJar = Files.createTempFile(absolutePath, "capsule_pathing_jar", ".jar");
+
+            final Path pathingJar = createTempFile(absolutePath, "capsule_pathing_jar", ".jar");
```

Comment:
I suppose the Files.createTempFile also check for file existence and generates a new file name if exists.
The new method does not have this fallback (and I think it should)

Reply:
arg indeed
will fix that